exe/floe

```diff
@@ -6,13 +6,21 @@ require "optimist"
 
 opts = Optimist.options do
   version("v#{Floe::VERSION}\n")
-  opt :workflow, "Path to your workflow json",                 :type => :string, :required => true
-  opt :input, "JSON payload to input to the workflow",         :default => '{}'
-  opt :credentials, "JSON payload with credentials",           :default => "{}"
-  opt :docker_runner, "Type of runner for docker images",      :default => "docker"
-  opt :docker_runner_options, "Options to pass to the runner", :type => :strings
+  opt :workflow, "Path to your workflow json",                           :type => :string, :required => true
+  opt :input, "JSON payload to input to the workflow",                   :default => '{}'
+  opt :credentials, "JSON payload with credentials",                     :default => "{}"
+  opt :docker_runner, "Type of runner for docker images",                :default => "docker", :short => 'r'
+  opt :docker_runner_options, "Options to pass to the runner",           :type => :strings, :short => 'o'
+  opt :docker,                "Use docker to run images  (shortcut)",    :type => :boolean, :short => 'd'
+  opt :podman,                "Use podman to run images  (shortcut)",    :type => :boolean, :short => 'p'
+  opt :kubernetes,            "Use kubernetes to run images (shortcut)", :type => :boolean, :short => 'k'
 end
 
+# alternative options
+opts[:docker_runner]="docker" if opts[:docker]
+opts[:docker_runner]="podman" if opts[:podman]
+opts[:docker_runner]="kubernetes" if opts[:kubernetes]
```

Comment:
```suggestion
opts[:docker_runner] = "docker" if opts[:docker]
opts[:docker_runner] = "podman" if opts[:podman]
opts[:docker_runner] = "kubernetes" if opts[:kubernetes]
```

Reply:
Do we want to handle the case where they specify both the short and the long option differently?  e.g. `floe --podman --docker-runner=kubernetes`
I'd expect either an error, or the long-option to take precedence where this looks like the short-hand will take precedence.

Reply:
I would prefer to just drop the long form

Personally, `--docker` is much easier than `--docker_runner=docker`. And it basically needs to be specified every time

Currently, the long form is defaulted so it will always be set.
I'll change the way it handles defaults and have long form override